service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/UploadC2DocumentsMidEventControllerTest.java

```diff
@@ -72,9 +73,9 @@ void shouldRemoveTemporaryC2DocumentForEmptyUrl() {
 
         AboutToStartOrSubmitCallbackResponse response = postMidEvent(CaseDetails.builder()
             .data(Map.of("temporaryC2Document", Map.of("document", Map.of())))
-            .build());
+            .build(), "get-fee");
 
-        assertThat(response.getData()).doesNotContainKey("temporaryC2Document");
+        assertThat((Map<String, Object>) response.getData().get("temporaryC2Document")).doesNotContainKey("document");
```

Comment:
can we do assertThat(response.getData()).extracting("temporyC2Document")... here?

Reply:
nice :+1: removed warning suppression thanks to that